test/test_minipyro.py

```diff
@@ -90,6 +90,59 @@ def model(data=None):
         assert data.shape == ()
 
 
+@pytest.mark.parametrize("backend", ["pyro", "minipyro", "funsor"])
+def test_rng_seed(backend):
+
+    def model(data=None):
+        loc = pyro.param("loc", torch.tensor(2.0))
+        scale = pyro.param("scale", torch.tensor(1.0))
+        x = pyro.sample("x", dist.Normal(loc, scale), obs=data)
+        return x
+
+    with pyro_backend(backend):
+        with handlers.seed(rng_seed=0):
+            expected = model()
+        with handlers.seed(rng_seed=0):
+            actual = model()
+        assert_close(actual, expected)
+
+
+@pytest.mark.parametrize("backend", ["pyro", "minipyro", "funsor"])
+def test_rng_state(backend):
+
+    def model(data=None):
+        loc = pyro.param("loc", torch.tensor(2.0))
+        scale = pyro.param("scale", torch.tensor(1.0))
+        x = pyro.sample("x", dist.Normal(loc, scale), obs=data)
+        return x
+
+    with pyro_backend(backend):
+        with handlers.seed(rng_seed=0):
+            model()
+            expected = model()
+        with handlers.seed(rng_seed=0):
+            model()
+            with handlers.seed(rng_seed=0):
+                model()
+            actual = model()
+        assert_close(actual, expected)
+
+
+@pytest.mark.parametrize("backend", ["pyro", "minipyro", "funsor"])
+def test_rng_state(backend):
```

Comment:
Is the rng_state being tested because of the default seed handler in `pyro_backend`?

Reply:
No, the rng_state is being tested irrelevant of the behavior of pyro_backend. I just wanted to add a test, and the natural place for that was in test_minipyro.py.

Reply:
It seems like there's a name clash with the test above?

Reply:
fixed.